### PR TITLE
docs: Add Platform-Compatibility matrix

### PR DESCRIPTION
**Pull request by nitkon**
Fixes: #136

Signed-off-by: Nitesh Konkar <niteshkonkar@in.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
         * [Image with systemd as init](#image-with-systemd-as-init)
         * [Image with the agent as init](#image-with-the-agent-as-init)
     * [Initrd creation](#initrd-creation)
+* [Platform-Distro Compatibility Matrix](#Platform-Distro-Compatibility-Matrix)
```
Comment (grahamwhaley) [inline README.md]: I think for the hotlink to work to the section you have to make all the words lowercase, like `#platform-distro-compatibility-matrix` - annoying though it is, I know...

File: README.md [hunk 2]
```diff
@@ -107,3 +108,7 @@ $ sudo -E PATH=$PATH make AGENT_INIT=yes initrd-only
 
 For further details,
 see [the initrd builder documentation](initrd-builder/README.md).
+
+## Platform-Distro Compatibility Matrix
+
+![Compatibility Matrix](images/platform-distro-compatibility-matrix.png)
```
Comment (grahamwhaley) [inline README.md]: Ah, I think @jodh-intel 's idea was to add the table in markdown, not as a binary image. Here is the raw md extracted from the other issue at: https://github.com/kata-containers/documentation/pull/186 :
```
 | | Alpine | CentOS | ClearLinux | EulerOS | Fedora | 
  |--|--|--|--|--|--|
  | **ARM64** | :heavy_check_mark: | :heavy_check_mark: |  | :heavy_check_mark: | :heavy_check_mark: | 
  | **PPC64le** | :heavy_check_mark: | :heavy_check_mark: |  |  | :heavy_check_mark: | 
  | **x86_64** | :heavy_check_mark: |:heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
```
which renders as

 | | Alpine | CentOS | ClearLinux | EulerOS | Fedora | 
  |--|--|--|--|--|--|
  | **ARM64** | :heavy_check_mark: | :heavy_check_mark: |  | :heavy_check_mark: | :heavy_check_mark: | 
  | **PPC64le** | :heavy_check_mark: | :heavy_check_mark: |  |  | :heavy_check_mark: | 
  | **x86_64** | :heavy_check_mark: |:heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |

**Comment by jodh-intel** [comment]
lgtm

**Comment by jodh-intel** [comment]
I think this PR falls into the exempt doc review category as there are no sentences to check:

- https://github.com/kata-containers/community/blob/master/Documentation-Review-Process.md#exempt

Also, this is a doc-only change which doesn't affect the CI.

**Comment by nitkon** [comment]
I guess it is good to merge then.

**Comment by grahamwhaley** [comment]
I think so - was just waiting on the CI's.
The Centos failed - but, nothing to do with this PR, so I will force merge this one.
For @chavafg @jcvenegas - here is the CI error I think:

```
[kata-containers-osbuilder-centos-7-4-PR] $ /bin/bash /tmp/jenkins3321864157334052621.sh
/bin/kata-collect-data.sh: line 481: losetup: command not found
/bin/kata-collect-data.sh: line 486: losetup: command not found
/tmp/jenkins/workspace/kata-containers-osbuilder-centos-7-4-PR /tmp/jenkins/workspace/kata-containers-osbuilder-centos-7-4-PR/go/src/github.com/kata-containers/tests
```

**Pull request merged**